tests/test_autoscaling/test_autoscaling.py

```diff
@@ -520,6 +520,43 @@ def test_describe_load_balancers():
     list(response['LoadBalancers']).should.have.length_of(1)
     response['LoadBalancers'][0]['LoadBalancerName'].should.equal('my-lb')
 
+@mock_autoscaling
+@mock_elb
+def test_create_elb_and_autoscaling_group_no_relationship():
+    INSTANCE_COUNT = 2
+    ELB_NAME = 'my-elb'
+
+    elb_client = boto3.client('elb', region_name='us-east-1')
+    _ = elb_client.create_load_balancer(
```

Comment:
Most of the Moto tests don't bother assigning any return value, not even to `_`

Reply:
(that is, when that value will not be used)

Reply:
Thats true, I suppose I copied/pasted that from somewhere?